frontend/src/views/auth/RegisterDone.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <auth-container>
-    <h1 class="display-1">Konto erstellen…</h1>
+    <h1 class="display-1">{{ $tc('auth.register.title') }}</h1>
```

Comment:
Key should be `auth.registerDone.title` (should probably be a new translation key in the JSON)